src/components/Good/Good.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { GoodShape } from '../../shapes/GoodShape';
+
+export const Good = ({ good, selectTodo, deleteTodo, selected }) => (
+  <div className="good">
+    <button
+      type="button"
+      className={selected.includes(good.title)
+        ? 'good__item selected' : 'good__item'}
```

Comment:
use className library
https://www.npmjs.com/package/classnames
also Misha recorder a video about it
https://www.youtube.com/watch?v=l6af8bEqP9E&feature=youtu.be